### ISSUE TITLE
TypeError: "n is undefined"

### ISSUE THREAD

**Issue by carortobos** [alephdata/aleph#983]
I am new to Aleph. I have been practicing with opensanctions and I've uploaded all that data into aleph. However, when I try to make searches or access datasets created by opensanctions, my browser starts loading the data but, suddenly, the browser turns blank, and it does not show any results. Frames such as lateral panels also disappear and cannot access to anything. This only happens once I have loaded Opensanctions data. This is what the browser console shows as an error:

```
TypeError: "n is undefined"
    value Country.jsx:28
    React 7
    unstable_runWithPriority scheduler.production.min.js:270
    React 4
    Redux 4
    Nf error-toast-middleware.js:28
    Redux 2
    d asyncActionCreator.js:23
    s runtime.js:45
    _invoke runtime.js:264
    t runtime.js:98
    s runtime.js:45
    t runtime.js:137
    t runtime.js:147
```
It is just an example. I'm getting almost 20 errors like this. Can anybody help me?

**Comment by pudo** [comment]
This is fixed in 3.4.7, can you confirm?

**Comment by pudo** [comment]
Closing this as it's been absent in several clean installs.
